com.gitlab.JakobDev.jdReplace.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.gitlab.JakobDev.jdReplace
+runtime: org.kde.Platform
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: "6.2"
+command: jdReplace
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_PYWEBENGINE=1
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc

```

Comment:
Are you sure you don't need here a filesystem access permission? I suggest adding `--filesystem=home`.